src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DateAdd.txt

```diff
@@ -126,10 +126,8 @@ DateTime(2000,1,1,0,0,0,0)
 >> DateAdd(DateTime(2000,1,1,0,0,0),-1,TimeUnit.Milliseconds)
 DateTime(1999,12,31,23,59,59,999)
 
-// TO BE DETERMINED: Adding fractional values
-// Subtracting 1.9 ms truncated to 1ms
-// >> DateAdd(DateTime(2000,1,1,0,0,0),-1.9,TimeUnit.Milliseconds)
-// DateTime(1999,12,31,23,59,59,999)
+>> DateAdd(DateTime(2000,1,1,0,0,0),-1.9,TimeUnit.Milliseconds)
```

Comment:
>-1.9 [](http://example.com/codeflow?start=36&length=4)

I'd also try with -1.4 and -1.5

Reply:
May add in a future PR; the value 1.9 is nice because it shows that:
- If we accept the decimal part, we don't round or truncate
- If we do accept the decimal part, we truncate, and not round up